RELEASES.md

```diff
@@ -1,3 +1,120 @@
+Version 1.50.0 (2021-02-11)
+============================
+
+Language
+-----------------------
+- [You can now use `const` values for `T` in `[T; N]` array expressions.][79270]
+- [Assignments to `ManuallyDrop<T>` union fields are now considered safe.][78068]
+
+Compiler
+-----------------------
+- [Added tier 3\* support for the `armv5te-unknown-linux-uclibceabi` target.][78142]
+- [Added tier 3 support for the `aarch64-apple-ios-macabi` target.][77484]
+- [The `x86_64-unknown-freebsd` is now built with the full toolset.][79484]
+
+\* Refer to Rust's [platform support page][forge-platform-support] for more
+information on Rust's tiered platform support.
+
+Libraries
+-----------------------
+
+- [`proc_macro::Punct` now implements `PartialEq<char>`.][78636]
+- [On Unix platforms, the `std::fs::File` type now has a "niche" of `-1`.][74699]
+  This value cannot be a valid file descriptor, and now means `Option<File>` takes
+  up the same amount of space as `File`.
+
+Stabilized APIs
+---------------
+
+- [`UnsafeCell::get_mut`]
+- [`bool::then`]
+- [`RefCell::take`]
+- [`btree_map::Entry::or_insert_with_key`]
+- [`hash_map::Entry::or_insert_with_key`]
+- [`slice::fill`]
+
+The following previously stable methods are now `const`.
+
+- [`IpAddr::is_ipv4`]
+- [`IpAddr::is_ipv6`]
+- [`Layout::size`]
+- [`Layout::align`]
+- [`Layout::from_size_align`]
+- `pow` for all integer types.
+- `checked_pow` for all integer types.
+- `saturating_pow` for all integer types.
+- `wrapping_pow` for all integer types.
+- `next_power_of_two` for all unsigned integer types.
+- `checked_power_of_two` for all unsigned integer types.
+
+Cargo
+-----------------------
+
+- [Added the `[build.rustc-workspace-wrapper]` option.][cargo/8976]
+  This option sets the `rustc` wrapper to use to build workspace members.
+- [`cargo:rerun-if-changed` will now, if provided a directory, scan the entire
+  contents of that directory for changes.][cargo/8973]
+- [Periods (`.`) are now allowed in feature names.][cargo/8932]
```

Comment:
I would recommend not including this.  It is not allowed on crates.io, and is only relevant to people who plan to never publish there (the restriction was recently added, and only one user had an issue with it).

```suggestion
```

Reply:
How would you feel about, instead of removing it, including a sentence saying they are still not allowed on crates.io? I feel that makes clear enough to keep it.

Reply:
Nothing has really changed, so I'm not sure it's worth noting.  All characters have historically been allowed, and they are still allowed.  In 1.49, a warning was added that we intend to restrict the syntax in the future (https://github.com/rust-lang/cargo/pull/8814).  One user reported they were using periods, so that particular part was reverted (and [backported to 1.49](https://github.com/rust-lang/cargo/pull/8943)).  So, in essence, nothing changed in this release.

Reply:
Ah okay, seems reasonable not to include then.